bin/index.js

```diff
@@ -213,8 +229,18 @@ module.exports = function(Aquifer, AquiferGitConfig) {
 
       // Commit changes.
       .then(function () {
+        var signature;
+
         Aquifer.console.log('Commit changes...', 'status');
-        var signature = git.Signature['default'](repo);
+
+        if (options.name && options.email) {
```

Comment:
Should we adjust the way that we handle missing properties in the extension system? This behavior makes sense too me but if it doesn't make sense or is inconvenient we should change it.